admin_guide/managing_networking.adoc

```diff
@@ -420,50 +421,82 @@ $ oc create -f <replication_controller>.json
 oc describe rc <replication_controller>
 ----
 
-[[multicast]]
+[[admin-guide-networking-multicast]]
 == Enabling Multicast
 
-Normally, multicast traffic between pods is disabled in {product-title}, but you
-can enable it on a per-project basis by setting an annotation on the project's
-corresponding NetNamespace object:
-
+[IMPORTANT]
+====
+Enabling multicast networking is a Technology Preview only. Technology
+Preview features are not supported with Red Hat production service level
```

Comment:
Maybe instead of saying "For more information..." in the part below, you can make the 1st reference to Technology Preview a link to the resource.